tools/crates/cli/src/commands/starknet/transfer_args.rs

```diff
@@ -113,9 +111,7 @@ impl CommandRunner<ToolApp, TransferArgs> for RunTransferArgs {
                 revision_number: 0,
                 revision_height: 0,
             },
-            timeout_timestamp_on_b: Timestamp {
-                timestamp: args.timeout_timestamp,
-            },
+            timeout_timestamp_on_b: Timestamp::from_nanoseconds(args.timeout_timestamp),
```

Comment:
```suggestion
            timeout_timestamp_on_b: Timestamp::from_nanoseconds(args.timeout_timestamp * 1_000_000_000),
```

Because, we are using unix timestamp at cairo.

Reply:
Done in b70f8769382dbbe20e72136e6dd95058270d64b6